more_itertools/more.py

```diff
@@ -4953,11 +4951,11 @@ def idft(Xarr):
 
     See :func:`dft` for the Discrete Fourier Transform.
     """
-    N = len(Xarr)
-    roots_of_unity = [e ** (n / N * tau * 1j) for n in range(N)]
-    for k in range(N):
-        coeffs = [roots_of_unity[k * n % N] for n in range(N)]
-        yield _complex_sumprod(Xarr, coeffs) / N
+    l = len(Xarr)
```

Comment:
NO!  The formula here is *always* expressed with a capital `N` and a lowercase `n`'